planner/nodes/report_nodes.py

```diff
@@ -0,0 +1,22 @@
+from .base_processing_node import BaseProcessingNode, ProcessingArtifact
+
+
+class ReportProcessingNode(BaseProcessingNode):
+    def __init__(self, available_artifacts, outputs):
+        super(ReportProcessingNode, self).__init__(available_artifacts, outputs)
+        self.outputs = outputs
+        self.fmt = 'json'
+
+    def get_artifacts(self):
+        datahub_type = 'derived/report'
+        resource_name = 'datapackage_report'
+        tabular_artifacts = [
+            artifact for artifact in self.available_artifacts
+            if artifact.datahub_type == 'source/tabular'
+        ]
+        output = ProcessingArtifact(
+            datahub_type, resource_name,
+            tabular_artifacts, [],
```

Comment:
We should have them as non-streamed dependencies

Reply:
@akariv why? We won't be able to pass tabulator options and validate remote data in that case, no?

Reply:
Hm... actually I'm wrong - we get it anyways, right?
Fixing